ilc/systemjs/systemjs-transform-loader.js

```diff
@@ -0,0 +1,34 @@
+(function (global) {
```

Comment:
I think you don't need to reinvent https://github.com/systemjs/systemjs/blob/master/dist/extras/transform.js here

Reply:
just use it from SystemJS bundle

Reply:
We have issues with fetching of CSS bundles by this module which does not handle them and it wants to eval them
So that is why we created ourselves plugin